src/librustdoc/core.rs

```diff
@@ -76,8 +72,12 @@ crate struct DocContext<'tcx> {
     /// See `collect_intra_doc_links::traits_implemented_by` for more details.
     /// `map<module, set<trait>>`
     crate module_trait_cache: RefCell<FxHashMap<DefId, FxHashSet<DefId>>>,
-    /// Fake empty cache used when cache is required as parameter.
+    /// This same cache is used throughout rustdoc, including in `render`.
     crate cache: Cache,
+    /// Used by `clean::inline` to tell if an item has already been inlined.
+    crate inlined: RefCell<FxHashSet<DefId>>,
```

Comment:
I think this RefCell can be removed as well; from what I can see, you only mutate it from functions that take `&mut DocContext`.

Reply:
Thanks, done.